packages/digitalproperty-app/lib/landRegistry.js

```
@@ -20,316 +20,212 @@
+    */
+    async updateForSale() {
         const METHOD = 'updateForSale';
-
-
-        return this.bizNetworkConnection.getAssetRegistry('net.biz.digitalPropertyNetwork.LandTitle')
-      .then((registry) => {
-
-          LOG.info(METHOD, 'Getting assest from the registry.');
-          return registry.get('LID:1148');
-
-      }).then((result) => {
-
-	      let factory        = this.businessNetworkDefinition.getFactory();
-	      let transaction    = factory.newTransaction('net.biz.digitalPropertyNetwork','RegisterPropertyForSale');
-		  transaction.title  = factory.newRelationship('net.biz.digitalPropertyNetwork', 'LandTitle', 'LID:1148');
-		  transaction.seller = factory.newRelationship('net.biz.digitalPropertyNetwork', 'Person', 'PID:1234567890');
-
-          LOG.info(METHOD, 'Submitting transaction');
-
-          return this.bizNetworkConnection.submitTransaction(transaction);
-      }) // and catch any exceptions that are triggered
-      .catch(function (error) {
-          LOG.error('LandRegsitry:updateForSale', error);
-          throw error;
-      });
+        try {
+            let registry = await this.bizNetworkConnection.getAssetRegistry('net.biz.digitalPropertyNetwork.LandTitle');
+            LOG.info(METHOD, 'Getting assest from the registry.');
+            let result = registry.get('LID:1148');
+            let factory        = this.businessNetworkDefinition.getFactory();
+	        let transaction    = factory.newTransaction('net.biz.digitalPropertyNetwork','RegisterPropertyForSale');
+		    transaction.title  = factory.newRelationship('net.biz.digitalPropertyNetwork', 'LandTitle', 'LID:1148');
+		    transaction.seller = factory.newRelationship('net.biz.digitalPropertyNetwork', 'Person', 'PID:1234567890');
+
+            LOG.info(METHOD, 'Submitting transaction');
+            await this.bizNetworkConnection.submitTransaction(transaction);
+        } catch(error) {
+        }
```

should we ignore the error?